libs/lb/lb-rs/src/service/logging.rs

```diff
@@ -42,7 +42,9 @@ pub fn init(config: &Config) -> LbResult<()> {
                 .with_target(false)
                 .with_filter(lockbook_log_level)
                 .with_filter(filter::filter_fn(|metadata| {
-                    metadata.target().starts_with("workspace")
+                    metadata.target().starts_with("lb_rs")
+                        || metadata.target().starts_with("dbrs")
+                        || metadata.target().starts_with("workspace")
```

Comment:
while desirable I believe this will make our CLI unusable. However I moved the entire cli config into core, and you could see if those flags are already suitable.

I think they're not, you'll probably want to add a `stdout_logs` flag and `false` it on cli.

but it will be nice to `true` it on the tests.